decidim-meetings/app/cells/decidim/meetings/dates_and_map/show.erb

```diff
@@ -19,7 +19,7 @@
 
   <%= static_map %>
 
-  <% if online? %>
+  <% if meeting.online? && meeting.iframe_access_level_allowed_for_user?(current_user) %>
```

Comment:
I think this check should actually be here:

https://github.com/decidim/decidim/blob/e075c4c628340cbdf982bcb5ed16fd90985250e7/decidim-core/app/cells/decidim/address/online.erb#L5

as in your screenshot you're hiding the full address (with the **Online** text) and that text should be displayed

Reply:
Understood. Will update and test.

Reply:
Question @andreslucena: If the condition should move to core, how come the functionality worked?

Reply:
These are the calls to this cell in the codebase: 

```erb
decidim-conferences/app/views/decidim/conferences/conference_program/_program_meeting.html.erb:  <%= cell("decidim/address", meeting) if translated_attribute(meeting.location).present? %>
decidim-core/app/views/decidim/shared/_static_map.html.erb:  <%= cell("decidim/address", geolocalizable) %>
decidim-core/spec/cells/decidim/address_cell_spec.rb:  let(:my_cell) { cell("decidim/address", model) }
decidim-meetings/app/cells/decidim/meetings/dates_and_map/show.erb:    <%= cell("decidim/address", meeting, online: true) %>
decidim-meetings/app/views/decidim/participatory_spaces/_conference_venues.html.erb:              <%== cell("decidim/address", meeting) %>
decidim-design/app/helpers/decidim/design/address_helper.rb:                  { name: "Address", url: "https://github.com/decidim/decidim/blob/develop/decidim-core/app/cells/decidim/address_cell.rb" }
```

Basically, this is used for displaying any kind of address of a resource, for instance when there are budgets with addresses (so an "offline" address). 

Seeing again the related code, another place for the spec could be `decidim-core/spec/cells/decidim/address_cell_spec.rb`

Reply:
Should I update to both cells then?